schema/spacewalk/upgrade/susemanager-schema-3.2.16-to-susemanager-schema-3.2.17/001-suseproductsccrepo-delete-invalid-entries.sql

```diff
@@ -0,0 +1,14 @@
+delete from suseproductsccrepository where channel_label = 'suse-openstack-cloud-6-pool-x86_64-sp1';
```

Comment:
I am copying this script to `susemanager-schema-4.0.7-to-susemanager-schema-4.0.8`

Should not happen, but it does not hurt, and anyway we don't have any other script, and we need it to tag.